ambry-rest/src/main/java/com.github.ambry.rest/RestServerMain.java

```diff
@@ -79,6 +80,9 @@ public static void main(String[] args) {
    * @throws IOException
    */
   private static SSLFactory getSSLFactoryIfRequired(VerifiableProperties verifiableProperties) throws Exception {
+    if (new ClusterMapConfig(verifiableProperties).clusterMapHttp2Enable) {
```

Comment:
Can we just create one instance of `ClusterMapConfig` ? No need to create ClusterMapConfig twice in some cases.